http-server/src/lib.rs

```diff
@@ -24,11 +24,12 @@
 // IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 // DEALINGS IN THE SOFTWARE.
 
+mod access_control;
 mod module;
 mod response;
 mod server;
 
-pub use jsonrpsee_utils::http::access_control::{AccessControl, AccessControlBuilder};
+pub use access_control::{AccessControl, AccessControlBuilder, AllowHosts, Host};
```

Comment:
these re-exports are needed to configure the AccessControl properly (i.e, non-default ^^)